certified-connectors/PDF4me Connect/apiDefinition.swagger.json

```diff
@@ -2461,6 +2461,315 @@
           }
         ]
       }
+    },
```

Comment:
@yispot Please remove this file from your PR as this is for another connector.

Reply:
Looks like this file is not in my DotToPDFConnector folder.